commit 56e97e70f7e36a678ef1847a0b0e2d46caa7ac40
Author: David S. Miller <davem@davemloft.net>
Date:   Sat Feb 16 13:44:39 2019 -0800

    net: Add header for usage of fls64()
    
    [ Upstream commit 8681ef1f3d295bd3600315325f3b3396d76d02f6 ]
    
    Fixes: 3b89ea9c5902 ("net: Fix for_each_netdev_feature on Big endian")
    Suggested-by: Eric Dumazet <eric.dumazet@gmail.com>
    Signed-off-by: David S. Miller <davem@davemloft.net>
    Signed-off-by: Sasha Levin <sashal@kernel.org>

diff --git a/include/linux/netdev_features.h b/include/linux/netdev_features.h
index fce28562bed2..4c76fe2c8488 100644
--- a/include/linux/netdev_features.h
+++ b/include/linux/netdev_features.h
@@ -11,6 +11,7 @@
 #define _LINUX_NETDEV_FEATURES_H
 
 #include <linux/types.h>
+#include <linux/bitops.h>
 #include <asm/byteorder.h>
 
 typedef u64 netdev_features_t;
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1456_56e97e70f7e36a678ef1847a0b0e2d46caa7ac40.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1456_56e97e70f7e36a678ef1847a0b0e2d46caa7ac40.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
